README.md

```diff
@@ -34,7 +34,6 @@ We **WILL** change this and add intermediaries in the future.
 After cloning the repository:
 
 ```
-$ go get -u github.com/go-swagger/go-swagger/cmd/swagger@v0.26.1
```

Comment:
moreved here @bobcallaway

Reply:
I swear I looked for this but totally missed it... sorry @cpanato :(